.travis.yml

```diff
@@ -5,5 +5,6 @@ php:
   - 5.6
   - 7.0
   - 7.1
+  - 8.1
```

Comment:
Travis should be removed and 8.1 should be added at https://github.com/samdark/yii2-psr-log-target/blob/master/.github/workflows/build.yml instead.

Reply:
Corected